app/src/main/java/com/example/unscramble/ui/theme/Color.kt

```diff
@@ -15,13 +15,69 @@
+val md_theme_light_secondaryContainer = Color(0xFFE0E1F9)
+val md_theme_light_onSecondaryContainer = Color(0xFF181A2C)
+val md_theme_light_tertiary = Color(0xFF77536D)
+val md_theme_light_onTertiary = Color(0xFFFFFFFF)
+val md_theme_light_tertiaryContainer = Color(0xFFFFD7F1)
+val md_theme_light_onTertiaryContainer = Color(0xFF2D1228)
+val md_theme_light_error = Color(0xFFBA1A1A)
+val md_theme_light_errorContainer = Color(0xFFFFDAD6)
+val md_theme_light_onError = Color(0xFFFFFFFF)
+val md_theme_light_onErrorContainer = Color(0xFF410002)
+val md_theme_light_background = Color(0xFFFEFBFF)
+val md_theme_light_onBackground = Color(0xFF1B1B1F)
+val md_theme_light_surface = Color(0xFFFEFBFF)
+val md_theme_light_onSurface = Color(0xFF1B1B1F)
+val md_theme_light_surfaceVariant = Color(0xFFE3E1EC)
+val md_theme_light_onSurfaceVariant = Color(0xFF46464F)
+val md_theme_light_outline = Color(0xFF767680)
+val md_theme_light_inverseOnSurface = Color(0xFFF3F0F4)
+val md_theme_light_inverseSurface = Color(0xFF303034)
+val md_theme_light_inversePrimary = Color(0xFFBAC3FF)
+val md_theme_light_shadow = Color(0xFF000000)
+val md_theme_light_surfaceTint = Color(0xFF4355B9)
+val md_theme_light_outlineVariant = Color(0xFFC7C5D0)
+val md_theme_light_scrim = Color(0xFF000000)
+
+val md_theme_dark_primary = Color(0xFFBAC3FF)
+val md_theme_dark_onPrimary = Color(0xFF08218A)
+val md_theme_dark_primaryContainer = Color(0xFF293CA0)
+val md_theme_dark_onPrimaryContainer = Color(0xFFDEE0FF)
+val md_theme_dark_secondary = Color(0xFFC3C5DD)
+val md_theme_dark_onSecondary = Color(0xFF2D2F42)
+val md_theme_dark_secondaryContainer = Color(0xFF434659)
+val md_theme_dark_onSecondaryContainer = Color(0xFFE0E1F9)
+val md_theme_dark_tertiary = Color(0xFFE6BAD7)
+val md_theme_dark_onTertiary = Color(0xFF44263D)
+val md_theme_dark_tertiaryContainer = Color(0xFF5D3C55)
+val md_theme_dark_onTertiaryContainer = Color(0xFFFFD7F1)
+val md_theme_dark_error = Color(0xFFFFB4AB)
+val md_theme_dark_errorContainer = Color(0xFF93000A)
+val md_theme_dark_onError = Color(0xFF690005)
+val md_theme_dark_onErrorContainer = Color(0xFFFFDAD6)
+val md_theme_dark_background = Color(0xFF1B1B1F)
+val md_theme_dark_onBackground = Color(0xFFE4E1E6)
+val md_theme_dark_surface = Color(0xFF1B1B1F)
+val md_theme_dark_onSurface = Color(0xFFE4E1E6)
+val md_theme_dark_surfaceVariant = Color(0xFF46464F)
+val md_theme_dark_onSurfaceVariant = Color(0xFFC7C5D0)
+val md_theme_dark_outline = Color(0xFF90909A)
+val md_theme_dark_inverseOnSurface = Color(0xFF1B1B1F)
+val md_theme_dark_inverseSurface = Color(0xFFE4E1E6)
+val md_theme_dark_inversePrimary = Color(0xFF4355B9)
+val md_theme_dark_shadow = Color(0xFF000000)
+val md_theme_dark_surfaceTint = Color(0xFFBAC3FF)
+val md_theme_dark_outlineVariant = Color(0xFF46464F)
+val md_theme_dark_scrim = Color(0xFF000000)
+
 
-val Purple40 = Color(0xFF6650a4)
-val PurpleGrey40 = Color(0xFF625b71)
-val Pink40 = Color(0xFF7D5260)
\ No newline at end of file
+val seed = Color(0xFF3F51B5)
```

Comment:
what's seed? is that a color provided by the template?

Reply:
I used the primary color as seed color in the Material theme builder and downloaded the files.